### ISSUE TITLE
Does not start after fresh install (missing dependencies)

### ISSUE THREAD

**Issue by gergles** [Nordaaker/convos#368]
CentOS 7

After installing perl-devel to get around a missing MakeMaker error, running the script/convos install script completes successfully. However, convos still complains about missing dependencies:

```
Is is not possible to start Convos at this point, since
there are some missing dependencies that need to be installed:

-  JSON::Validator 0.91 # Can't locate Digest/MD5.pm
-  LinkEmbedder 0.05 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojo::IOLoop::ForkCall 0.17 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojo::IRC 0.44 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious 7.29 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious::Plugin::AssetPack 1.41 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious::Plugin::OpenAPI 1.06 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require

Run "script/convos install" to install the missing dependencies above,
or run "CONVOS_DEBUG=1 script/convos daemon --listen http://*:8000" for more information.
```

(There's also a typo in the error message... "is is" instead of "it is".)

Installing perl-Digest-MD5 from yum solved the problem, but then it complained about missing Digest/SHA.pm. Installing also perl-Digest-SHA finally fixed it.

The install script should check for every dependency and install them through CPAN. It looks like these are missing from the checks (perhaps an assumption they are always present?)

tl;dr: To install on CentOS 7, first yum install perl-devel perl-Digest-MD5 perl-Digest-SHA before running the convos installer.

**Comment by marcusramberg** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @jhthorsen.

Possibly related issues are https://github.com/Nordaaker/convos/issues/232 (Convos will not start), https://github.com/Nordaaker/convos/issues/198 (Install.sh), https://github.com/Nordaaker/convos/issues/314 (Install convos), https://github.com/Nordaaker/convos/issues/130 ([ missing in nicks), and https://github.com/Nordaaker/convos/issues/305 (Does not support "away" ?).

**Comment by pink-mist** [comment]
You do not have a full perl install. Digest::MD5 is a part of perl, yet your error message is complaining that it can't find it. Centos and RedHat including Fedora (until the very latest version) does not have a complete perl installation in either the ```perl``` or the ```perl-devel``` packages. You need to install the ```perl-core``` package which is what every single other distribution refers to as the ```perl``` package.

Thankfully in the very latest Fedora version, they did rename their ```perl-core``` package to ```perl```, and their old so-called ```perl``` package was renamed to ```perl-interpreter```. Hopefully this should be fixed in the next version of RHEL, and then in that subsequent CentOS version as well.
